cannon/mipsevm/tests/evm_common_test.go

```diff
@@ -20,7 +20,9 @@ import (
 	"github.com/ethereum-optimism/optimism/cannon/mipsevm/testutil"
 )
 
-func TestEVM(t *testing.T) {
+func TestEVM_OpenMIPS(t *testing.T) {
+	testutil.Cannon32OnlyTest(t, "Skipping MIPS32 assembly test vectors for cannon64")
```

Comment:
Are these OpenMIPS tests are failing because they're compiled for 32-bit?  Should we add a follow-up ticket that compiles them for 64-bit?

Reply:
They are failing for various reasons, including 64-bit misaligned return addresses and mips32 syscalls.

However, I don't think it's worth it to port these to 64-bits. The diff tests are more comprehensive than openmips. So once mips64 is the default vm, we should delete the openmips test vectors.